shared/actions/config/index.tsx

```diff
@@ -718,7 +713,7 @@ const gregorPushState = (action: GregorGen.PushStatePayload) => {
   const lastSeenItem = items.find(i => i.item && i.item.category === 'whatsNewLastSeenVersion')
   if (lastSeenItem) {
     const {body} = lastSeenItem.item
-    const pushStateLastSeenVersion = body.toString()
+    const pushStateLastSeenVersion = Buffer.from(body).toString()
```

Comment:
turns out we need to explicitly cast this. the old code assumed when body is decoded we get a buffer but the actual api states we get a `UInt8Array` (which doesn't have `toString`). We do have some code paths where we accidentally got a `Buffer` back (which is a subclass of `UInt8Array` but this wasn't ever by design